packages/component-model-manuscript/entities/manuscript/index.js

```diff
@@ -324,6 +324,7 @@ class Manuscript extends BaseModel {
     const teamsToClean = ['opposedReviewers', 'suggestedReviewers']
 
     /* eslint no-param-reassign: ["error", { "props": false }] */
+    input.author.email = input.author.email.trim()
```

Comment:
Are we happy there will always be an email property on author to call `trim` on?

Reply:
:thinking: fair point - I'll change